frontend/src/lib/Status.tsx

```diff
@@ -1,7 +1,17 @@
+import { NavigateFunction } from 'react-router-dom';
+
+export const RouteOnNotOK = (responce: Response, navigate: NavigateFunction) => {
```

Comment:
```suggestion
export const RouteOnNotOK = (response: Response, navigate: NavigateFunction) => {
```
I would suggest you install a spell checker tool